errors/errors-controller.js

```diff
@@ -0,0 +1,18 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    console.log('custom err>>>>>>', err)
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
+
+exports.handlePsqlErrors = (err, req, res, next) => {
+    console.log('PSQL err>>>>>>', err)
+  // if (err.code === '23502') {
+  //   res.status(400).send({ msg: 'Bad request' });
+  // }
+};
+
+exports.handleServerErrors = (err, req, res, next) => {
+  console.log('server err>>>>>>', err);
```

Comment:
This console log on line `16` should be the only one you need, in the other error handlers the error is being 'handled' and so it won't need to be logged on lines `2` and `9`. However if the error gets here then it is 'unhandled' so it is useful to get a log of it